examples/step-79/doc/intro.dox

```diff
@@ -103,60 +103,54 @@ following:
 
   \nabla \cdot \boldsymbol{\sigma}(\rho) + \mathbf{F} = 0 \quad \text{on } \Omega
 @f]
-The final constraint, the balance of linear momentum given by the elasticity equation,
+The final constraint, the balance of linear momentum (sometimes referred to as the elasticity equation),
```

Comment:
This is the one line that I do feel quite strongly about. I've got quite a number of books on solid mechanics, and I don't see this phrase appearing in any of them. I think that its fine if you use it, but I'd strongly suggest that its first and foremost introduced as the balance of linear momentum (as it's named in the canonical texts for both continuum mechanics and solid mechanics), and then you introduce this alternative phrase (which is maybe something that mathematicians use more often?).

Reply:
Absolutely! I do not have any books that I know of on solid mechanics, so I'll go with your phrasing. :)